pkgs/by-name/li/libguestfs-appliance-debian/package.nix

```diff
@@ -0,0 +1,113 @@
+{
+  lib,
+  stdenvNoCC,
+  vmTools,
+  fetchurl,
+  runCommand,
+  qemu-utils,
+}:
+let
+  debianSnapshot = "20250227T210548Z";
+
+  debianArch =
+    {
+      "x86_64-linux" = "amd64";
+      "aarch64-linux" = "arm64";
+    }
+    .${stdenvNoCC.hostPlatform.system};
```

Comment:
same as below

Reply:
Isn't that OK since `meta.platforms` is restricted to these two? When trying to build for a different architecture I get the expected "unsupported system" error.